content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -190,7 +179,9 @@ configuration before deploying Kubeflow:
 
 ## Post Installation
 
-Kubeflow 0.6 release brings multi-tenancy support and user are not able to create notebooks in `kubeflow`, `default` namespace. Instead, please create a `Profile` using `kubectl apply -f profile.yaml` and profile controller will create new namespace and service account which is allowed to create notebook in that namespace.
+Kubeflow 0.6 release brings multi-tenancy support and user are not able to create notebooks in `kubeflow`, `default` namespace.
+
+The first time you visit the cluster, you can ceate a namespace `anonymous` to use. If you want to create different users, you can create `Profile` and then `kubectl apply -f profile.yaml`. Profile controller will create new namespace and service account which is allowed to create notebook in that namespace.
```

Comment:
Would it help to link to these docs for more info?
https://www.kubeflow.org/docs/components/multi-tenancy/

Reply:
That's a good point. Link the docs.